airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -833,6 +835,127 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+  JwtAuthenticator:
+    title: JWT Authenticator
+    description: Authenticator for requests using JWT authentication flow.
+    type: object
+    required:
+      - type
+      - secret_key
+      - algorithm
+    properties:
+      type:
+        type: string
+        enum: [JwtAuthenticator]
+      secret_key:
+        type: string
+        description: Secret used to sign the JSON web token.
+        examples:
+          - "{{ config['secret_key'] }}"
+      base64_encode_secret_key:
+        type: boolean
+        description: When set to true, the secret key will be base64 encoded prior to being encoded as part of the JWT. Only set to "true" when required by the API.
+        default: False
+      algorithm:
+        type: string
+        description: Algorithm used to sign the JSON web token.
+        enum:
+          [
+            "HS256",
+            "HS384",
+            "HS512",
+            "ES256",
+            "ES256K",
```

Comment:
this enum value isn't in [jwt.py](https://github.com/airbytehq/airbyte/pull/37005/files#diff-869911f7176fe3fef7e2cacd11b92a24d1b2537ee4f5225f7ff96c37dda97c0eR22-R33)